config/cscs.py

```diff
@@ -201,6 +201,13 @@ class ReframeSettings:
                     'cxx': 'mpicxx',
                     'ftn': 'mpif90',
                 },
+                'PrgEnv-gdr': {
+                    'type': 'ProgEnvironment',
+                    'modules': ['gmvapich2/17.02_cuda_8.0_gdr'],
```

Comment:
@victorusu Don't we need here the `PrgEnv-gnu` as well?

Reply:
I have asked @jgphpc because initially I have put `PrgEnv-gnu` but if you do so you have module conflicts. It is a different environment.